week-04/README.md

```diff
@@ -15,61 +15,79 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 7, Problem 3]
+- *Question* (aamiri2-stat6250): While creating a format with a VALUE statement, why is it important to have the correct format in the beginning and ending statements?
 
 
 
 [Course Textbook Chapter 7, Problem 4]
+- *Question* (aamiri2-stat6250): How will we be able to use the values assigned to each variable during the analysis? Are values easier to use as compared to the actual variable name?
 
 
 
 [Course Textbook Chapter 7, Problem 5]
+- *Question* (aamiri2-stat6250): Are there any differences in formatting a range VALUE statement between numeric and character values? 
 
 
 
-[Course Textbook Chapter 7, Problem 6]
+ [Course Textbook Chapter 7, Problem 6]
+- *Question* (aamiri2-stat6250): What is the result of going over the character limit in a label?
 
 
 
 [Course Textbook Chapter 7, Problem 7]
+- *Question* (aamiri2-stat6250): The textbook mentions you can use the keyword OTHER to label a missing value, are there any other keywords that can have the same function?
 
 
 
-[Course Textbook Chapter 7, Problem 8]
+ [Course Textbook Chapter 7, Problem 8]
```

Comment:
Existing content should not be edited or deleted in any way; please update PR and DM me in Slack with PR URL when ready for re-evaluation